### ISSUE TITLE
[Bug]: Failing to download podcast episode for specific podcast

### ISSUE THREAD

**Issue by chicknwinner** [advplyr/audiobookshelf#2042]
### Describe the issue

errors manually downloading an episode
`
[FfmpegHelpers] downloadPodcastEpisode: Cmd: ffmpeg -i pipe:0 -y -c copy -metadata podcast=1 -metadata album=Election Profit Makers -metadata album-sort=Election Profit Makers -metadata artist=Jon Kimball & David Rees & Starlee Kine -metadata artist-sort=Jon Kimball & David Rees & Starlee Kine -metadata comment=<p>Hello, this is David and I don't know what goes here so I'm gonna leave it blank. </p> -metadata subtitle=Kid Midas flies solo. -metadata disc=3 -metadata genre=Politics;Podcasts;News;Comedy -metadata language=en -metadata MVNM=Election Profit Makers -metadata title=Episode 224: EPM Mugshot Review -metadata title-sort=Episode 224: EPM Mugshot Review -metadata year=2023 -metadata date=Tue, 29 Aug 2023 22:00:00 -0400 -metadata releasedate=Tue, 29 Aug 2023 22:00:00 -0400 -metadata itunes-id=1136826256 -metadata podcast-type=episodic /podcasts/Election Profit Makers/Episode 224 - EPM Mugshot Review.mp3
`
[

> FfmpegHelpers] downloadPodcastEpisode: Error Error: ffmpeg exited with code 1: pipe:0: Function not implemented ffmpeg version 6.0 Copyright (c) 2000-2023 the FFmpeg developers built with gcc 12.2.1 (Alpine 12.2.1_git20220924-r10) 20220924 configuration: --prefix=/usr --disable-librtmp --disable-lzma --disable-static --disable-stripping --enable-avfilter --enable-gnutls --enable-gpl --enable-libaom --enable-libass --enable-libbluray --enable-libdav1d --enable-libdrm --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libmp3lame --enable-libopenmpt --enable-libopus --enable-libplacebo --enable-libpulse --enable-librist --enable-libsoxr --enable-libsrt --enable-libssh --enable-libtheora --enable-libv4l2 --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxcb --enable-libxml2 --enable-libxvid --enable-libzimg --enable-libzmq --enable-lto --enable-pic --enable-postproc --enable-pthreads --enable-shared --enable-vaapi --enable-vdpau --enable-vulkan --optflags=-O3 --enable-libjxl --enable-libsvtav1 --enable-libvpl libavutil 58. 2.100 / 58. 2.100 libavcodec 60. 3.100 / 60. 3.100 libavformat 60. 3.100 / 60. 3.100 libavdevice 60. 1.100 / 60. 1.100 libavfilter 9. 3.100 / 9. 3.100 libswscale 7. 1.100 / 7. 1.100 libswresample 5. 0.100 / 5. 0.100 libpostproc 57. 1.100 / 57. 1.100 pipe:0: Function not implemented 

### Steps to reproduce the issue

if i create a fresh docker image using ghcr.io/advplyr/audiobookshelf and add the podcast Election Profit Makers nad try to manually download Episode 224 it fails

### Audiobookshelf version

2.3.3

### How are you running audiobookshelf?

Docker

**Comment by advplyr** [comment]
I was able to reproduce this with that podcast using the docker container. When running locally I wasn't able to reproduce it so this may have to do with the ffmpeg build, but just making a guess there.

Have you found any other podcasts this happens for?

**Comment by chicknwinner** [comment]
I've had it happen before with different podcasts but there are no others that i can reproduce at the moment.

I have even been able to get that episode to download ok, but not consistently, by enabling/disabling scheduled downloads and retrying to manually download it. sometimes it just randomly works.

**Comment by chicknwinner** [comment]
```
2023-09-01 11:40:22

DEBUG

[podcastUtils] getPodcastFeed for "https://feeds.fireside.fm/epm/rss"

2023-09-01 11:40:23

DEBUG

[podcastUtils] getPodcastFeed for "https://feeds.fireside.fm/epm/rss" success - parsing xml

2023-09-01 11:40:25

DEBUG

[Watcher] Ignoring directory "/podcasts/Election Profit Makers"

2023-09-01 11:40:25

DEBUG

[FfmpegHelpers] downloadPodcastEpisode: Cmd: ffmpeg -i pipe:0 -y -c copy -metadata podcast=1 -metadata album=Election Profit Makers -metadata album-sort=Election Profit Makers -metadata artist=Jon Kimball & David Rees & Starlee Kine -metadata artist-sort=Jon Kimball & David Rees & Starlee Kine -metadata comment=<p>Hello, this is David and I don't know what goes here so I'm gonna leave it blank. </p> -metadata subtitle=Kid Midas flies solo. -metadata disc=3 -metadata genre=Politics;Podcasts;News;Comedy -metadata language=en -metadata MVNM=Election Profit Makers -metadata title=Episode 224: EPM Mugshot Review -metadata title-sort=Episode 224: EPM Mugshot Review -metadata year=2023 -metadata date=Tue, 29 Aug 2023 22:00:00 -0400 -metadata releasedate=Tue, 29 Aug 2023 22:00:00 -0400 -metadata itunes-id=1136826256 -metadata podcast-type=episodic /podcasts/Election Profit Makers/Episode 224 - EPM Mugshot Review.mp3

2023-09-01 11:40:26

ERROR

[FfmpegHelpers] downloadPodcastEpisode: Error Error: ffmpeg exited with code 1: pipe:0: Function not implemented ffmpeg version 6.0 Copyright (c) 2000-2023 the FFmpeg developers built with gcc 12.2.1 (Alpine 12.2.1_git20220924-r10) 20220924 configuration: --prefix=/usr --disable-librtmp --disable-lzma --disable-static --disable-stripping --enable-avfilter --enable-gnutls --enable-gpl --enable-libaom --enable-libass --enable-libbluray --enable-libdav1d --enable-libdrm --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libmp3lame --enable-libopenmpt --enable-libopus --enable-libplacebo --enable-libpulse --enable-librist --enable-libsoxr --enable-libsrt --enable-libssh --enable-libtheora --enable-libv4l2 --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxcb --enable-libxml2 --enable-libxvid --enable-libzimg --enable-libzmq --enable-lto --enable-pic --enable-postproc --enable-pthreads --enable-shared --enable-vaapi --enable-vdpau --enable-vulkan --optflags=-O3 --enable-libjxl --enable-libsvtav1 --enable-libvpl libavutil 58. 2.100 / 58. 2.100 libavcodec 60. 3.100 / 60. 3.100 libavformat 60. 3.100 / 60. 3.100 libavdevice 60. 1.100 / 60. 1.100 libavfilter 9. 3.100 / 9. 3.100 libswscale 7. 1.100 / 7. 1.100 libswresample 5. 0.100 / 5. 0.100 libpostproc 57. 1.100 / 57. 1.100 pipe:0: Function not implemented

2023-09-01 11:40:26

DEBUG

[Watcher] No longer ignoring directory "/podcasts/Election Profit Makers"

2023-09-01 11:40:44

DEBUG

[podcastUtils] getPodcastFeed for "https://feeds.fireside.fm/epm/rss"

2023-09-01 11:40:44

DEBUG

[podcastUtils] getPodcastFeed for "https://feeds.fireside.fm/epm/rss" success - parsing xml

2023-09-01 11:40:46

DEBUG

[Watcher] Ignoring directory "/podcasts/Election Profit Makers"

2023-09-01 11:40:47

DEBUG

[FfmpegHelpers] downloadPodcastEpisode: Cmd: ffmpeg -i pipe:0 -y -c copy -metadata podcast=1 -metadata album=Election Profit Makers -metadata album-sort=Election Profit Makers -metadata artist=Jon Kimball & David Rees & Starlee Kine -metadata artist-sort=Jon Kimball & David Rees & Starlee Kine -metadata comment=<p>Hello, this is David and I don't know what goes here so I'm gonna leave it blank. </p> -metadata subtitle=Kid Midas flies solo. -metadata disc=3 -metadata genre=Politics;Podcasts;News;Comedy -metadata language=en -metadata MVNM=Election Profit Makers -metadata title=Episode 224: EPM Mugshot Review -metadata title-sort=Episode 224: EPM Mugshot Review -metadata year=2023 -metadata date=Tue, 29 Aug 2023 22:00:00 -0400 -metadata releasedate=Tue, 29 Aug 2023 22:00:00 -0400 -metadata itunes-id=1136826256 -metadata podcast-type=episodic /podcasts/Election Profit Makers/Episode 224 - EPM Mugshot Review.mp3

2023-09-01 11:40:48

DEBUG

[FfmpegHelpers] downloadPodcastEpisode: Complete

2023-09-01 11:40:49

INFO

[LibraryItem] Library item "7875936c-8116-4156-8254-e4259ba1c1fe" updated

2023-09-01 11:40:49

INFO

[PodcastManager] Successfully downloaded podcast episode "Episode 224: EPM Mugshot Review"

2023-09-01 11:40:49

DEBUG

[Watcher] No longer ignoring directory "/podcasts/Election Profit Makers"

2023-09-01 11:40:50

DEBUG

[Watcher] File Added /podcasts/Election Profit Makers/Episode 224 - EPM Mugshot Review.mp3

2023-09-01 11:40:50

DEBUG

[Watcher] Modified file in library "poo" and folder "0d8ad30f-fffc-4dd1-bba4-5e7ad20cefd6" with relPath "/Election Profit Makers/Episode 224 - EPM Mugshot Review.mp3"

2023-09-01 11:40:54

INFO

[Server] 1 Files Changed

2023-09-01 11:40:54

DEBUG

[Scanner] Scanning file update groups in folder "0d8ad30f-fffc-4dd1-bba4-5e7ad20cefd6" of library "poo"

2023-09-01 11:40:54

DEBUG

[Scanner] scanFolderUpdates fileUpdateGroup [object Object]

2023-09-01 11:40:54

DEBUG

[Scanner] Folder update for relative path "Election Profit Makers" is in library item "Election Profit Makers" - scan for updates

2023-09-01 11:40:55

DEBUG

Library Item "/podcasts/Election Profit Makers" Media file scan took 143ms with 1 audio files averaging 143ms per MB

2023-09-01 11:40:55

INFO

[LibraryItem] Library item "7875936c-8116-4156-8254-e4259ba1c1fe" updated

2023-09-01 11:40:55

DEBUG

[Scanner] Folder scan results [object Object]

2023-09-01 11:41:00

DEBUG

[CronManager] Start executing podcast cron * * * * * for 1 item(s)

2023-09-01 11:41:00

INFO

[PodcastManager] runEpisodeCheck: "Election Profit Makers" | Last check: Fri Sep 01 2023 11:40:01 GMT+1000 (Australian Eastern Standard Time) | Latest episode pubDate: Wed Aug 30 2023 12:00:00 GMT+1000 (Australian Eastern Standard Time)

2023-09-01 11:41:00

DEBUG

[PodcastManager] runEpisodeCheck: "Election Profit Makers" checking for episodes after Wed Aug 30 2023 12:00:00 GMT+1000 (Australian Eastern Standard Time)

2023-09-01 11:41:00

DEBUG

[podcastUtils] getPodcastFeed for "https://feeds.fireside.fm/epm/rss"

2023-09-01 11:41:01

DEBUG

[podcastUtils] getPodcastFeed for "https://feeds.fireside.fm/epm/rss" success - parsing xml

2023-09-01 11:41:01

DEBUG

[PodcastManager] runEpisodeCheck: 0 episodes found

2023-09-01 11:41:01

DEBUG

[PodcastManager] No new episodes for "Election Profit Makers"

2023-09-01 11:41:01

DEBUG

[CronManager] Finished executing podcast cron * * * * * for 1 item(s)

2023-09-01 11:42:00

DEBUG

[CronManager] Start executing podcast cron * * * * * for 1 item(s)

2023-09-01 11:42:00

INFO

[PodcastManager] runEpisodeCheck: "Election Profit Makers" | Last check: Fri Sep 01 2023 11:41:01 GMT+1000 (Australian Eastern Standard Time) | Latest episode pubDate: Wed Aug 30 2023 12:00:00 GMT+1000 (Australian Eastern Standard Time)

2023-09-01 11:42:00

DEBUG

[PodcastManager] runEpisodeCheck: "Election Profit Makers" checking for episodes after Wed Aug 30 2023 12:00:00 GMT+1000 (Australian Eastern Standard Time)

2023-09-01 11:42:00

DEBUG

[podcastUtils] getPodcastFeed for "https://feeds.fireside.fm/epm/rss"

2023-09-01 11:42:00

DEBUG

[podcastUtils] getPodcastFeed for "https://feeds.fireside.fm/epm/rss" success - parsing xml

2023-09-01 11:42:00

DEBUG

[PodcastManager] runEpisodeCheck: 0 episodes found

2023-09-01 11:42:00

DEBUG

[PodcastManager] No new episodes for "Election Profit Makers"

2023-09-01 11:42:00

DEBUG

[CronManager] Finished executing podcast cron * * * * * for 1 item(s)
```

**Comment by advplyr** [comment]
Is this still an issue on the current release 2.9.0? A lot has changed since
